deepmd/pt/model/model/__init__.py

```
@@ -280,6 +283,8 @@ def get_standard_model(model_params):
         modelcls = DOSModel
     elif fitting_net["type"] in ["ener", "direct_force_ener"]:
         modelcls = EnergyModel
+        if "hessian_mode" in model_params.keys() and model_params["hessian_mode"]:
```

Ruff

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)